src/data/community-meetups.json

```
@@ -424,5 +424,11 @@
     "emoji": ":belgium:",
     "location": "Brussels",
     "link": "https://dao.brussels/"
+  },
+  {
+    "title": "Ethereum Edmonton Meetup",
+    "emoji": ":canada:",
+    "location": "Edmonton",
+    "link": "https://www.meetup.com/es/ethereum-edmonton-meetup/"
```

Remove lang specific path (let browser handle it)

```suggestion
    "link": "https://www.meetup.com/ethereum-edmonton-meetup/"
```